Kickstarter-iOS/Views/PaymentMethodsFooterView.swift

```diff
@@ -0,0 +1,39 @@
+import Library
+import Prelude
+import UIKit
+
+internal protocol PaymentMethodsFooterViewDelegate: class {
+  func didTapAddNewCardButton()
```

Comment:
I think we should stick a convention for our delegate methods. They should always describe the object that's triggering the action as well as pass in that object and any additional values, something like:

```swift
func paymentMethodsFooterViewDidTapAddNewCardButton(_ footerView: PaymentMethodsFooterView)
```

More here: https://google.github.io/swift/#delegate-methods